### ISSUE TITLE
[FEATURE REQUEST] Add option to resume playback on next start up.

### ISSUE THREAD

**Issue by SubhadeepJasu** [artemanufrij/playmymusic#157]
The player already has a feature to resume the previously played song before the player was closed. Just add a toggle in the app menu to turn that feature on / off.

**Comment by artemanufrij** [comment]
Do you mean Melody should instand resume playing on startup?

**Comment by SubhadeepJasu** [comment]
No, I mean its okay the way the resume work rn. Just add an option to disable the feature or enable it. Basically choosing between. Starting the player fresh when the play button plays the music from beginning. OR the way it is rn where the music resumes from the time were it was left.

**Comment by artemanufrij** [comment]
Okay... Let me a couple hours

**Comment by SubhadeepJasu** [comment]
Basically like this:  `(  O)  Resume playback from the time you left it` something down that line. So if it is enabled then hitting play button will resume the music from the time it was left. If it is disabled then hitting play button will start the music from beginning. You get it.

**Comment by artemanufrij** [comment]
@SubhadeepJasu could you test current master branch please?

**Comment by SubhadeepJasu** [comment]
sure
